test/spec/core/ElementRegistrySpec.js

```diff
@@ -206,6 +206,62 @@ describe('ElementRegistry', function() {
 
   });
 
+
+  describe('find', function() {
```

Comment:
Should we maybe also add a test case, what happens, when no match was found?

Reply:
Sure, I've added one additional test case.